src/client/java/teammates/client/scripts/sql/DataMigrationForCourseEntitySql.java

```diff
@@ -329,7 +330,7 @@ private teammates.storage.sqlentity.FeedbackSession createFeedbackSession(teamma
                 oldSession.getFeedbackSessionName(),
                 newCourse,
                 oldSession.getCreatorEmail(),
-                oldSession.getInstructions(),
+                SanitizationHelper.sanitizeForRichText(oldSession.getInstructions()),
```

Comment:
I think sanitization is already done in the feedback session constructor [here](https://github.com/TEAMMATES/teammates/blob/master/src/main/java/teammates/storage/sqlentity/FeedbackSession.java#L258)

Reply:
Reverted. Thanks!